Samples/Widgets/README.md

```diff
@@ -28,20 +28,51 @@ These samples demonstrate how to author Windows Widgets with WindowsAppSDK.
 * Open the solution file (`.sln`) from the subfolder of your preferred sample in Visual Studio.
 * From Visual Studio, either **Start Without Debugging** (Ctrl+F5) or **Start Debugging** (F5).
 
-## SingleWidget and MultpleWidgets sample
+## WidgetProvider sample
 
-Inside this sample folder you will find 2 different projects:
-- Sample project that demonstrates authoring of only 1 widget within the same app.
-- Sample project that demonstrates authoring >=1 widgets within the same app.
-  
-To get familiar with the implementation steps and API details it's recommended to start with the single widget authoring sample.
+This project includes a sample of a 3rd party widget provider. At this point, it's only possible to author widgets in C++ and Widgetprovider application must be a packaged application.
 
-Once the widgets ideology and implementation steps are understood, you may examine the MultipleWidgets sample to get an idea on how to create an application that will be able to serve multiple widgets at a time.
+## Converting sample from Console to Windows application
 
-Relevant documentation and full walkthrough of a similar sample may be found [here](https://learn.microsoft.com/en-us/windows/apps/develop/widgets/implement-widget-provider-win32).
+If the widget providing application doesn't have any UI - it's a good idea to convert the app to windows application so the console doesn't show to the end user.
+These are the required steps to convert the sample to windows application:
+- Right click on the SampleWidgetProviderApp in Visual Studio and open Properties. Navigate to Linker -> System and change SubSystem from Console to Windows. It can also be done by adding `<SubSystem>Console</SubSystem>` to the `<Link>..</Link>` section of the .vcxproj.
```

Comment:
this is saying "set it to Console", I think you meant to say Windows instead.

Reply:
```suggestion
- Right click on the SampleWidgetProviderApp in Visual Studio and open Properties. Navigate to Linker -> System and change SubSystem from Console to Windows. It can also be done by adding `<SubSystem>Windows</SubSystem>` to the `<Link>..</Link>` section of the .vcxproj.
```